static/app/components/lazyLoad.tsx

```diff
@@ -31,17 +37,24 @@ type Props<C extends ComponentType> = React.ComponentProps<C> & {
  * LazyLoad is used to dynamically load codesplit components via a `import`
  * call. This is primarily used in our routing tree.
  *
- * <LazyLoad component={() => import('./myComponent')} someComponentProps={...} />
+ * Outside the render path
+ * Const LazyComponent = lazy(() => import('./myComponent'))
+ *
+ * <LazyLoad LazyComponent={LazyComponent} someComponentProps={...} />
  */
 function LazyLoad<C extends ComponentType>({
   component,
   loadingFallback,
+  LazyComponent,
   ...props
 }: Props<C>) {
-  const LazyComponent = useMemo(
-    () => lazy<C>(() => retryableImport(component)),
```

Comment:
this is what i'm trying to avoid, a new lazy() every time the component is rendered